lib/ruby_lsp/ruby_lsp_rails/server.rb

```diff
@@ -380,17 +380,21 @@ def resolve_association_target(params)
         association_klass = const.reflect_on_association(params[:association_name].intern).klass
         source_location = Object.const_source_location(association_klass.to_s)
 
+        raise "Could not determine source location" unless source_location
+
         { location: source_location.first + ":" + source_location.second.to_s }
       rescue NameError
         nil
       end
 
+      #: (const: Module?) -> bool
       def active_record_model?(const)
         !!(
           const &&
             defined?(ActiveRecord) &&
             const.is_a?(Class) &&
             ActiveRecord::Base > const && # We do this 'backwards' in case the class overwrites `<`
+            const < ActiveRecord::Base && # this is just to narrow the type for Sorbet
```

Comment:
Do you need both?

```suggestion
            const < ActiveRecord::Base && # this is just to narrow the type for Sorbet
```

Reply:
Oh I see the `# We do this 'backwards' in case the class overwrites <`, is it why we do it twice?

Reply:
Yes. (introduced in https://github.com/Shopify/ruby-lsp-rails/pull/328)